usr/share/rear/verify/DP/default/400_verify_dp.sh

```diff
@@ -8,9 +9,9 @@ StopIfError "Data Protector Cell Manager not set in /etc/opt/omni/client/cell_se
 
 if test "$PING" ; then
 	ping -c 1 "${CELL_SERVER}" >/dev/null 2>&1
-	StopIfError "Sorry, but cannot reach Data Protector Cell Manager ${CELL_SERVER}"
+	StopIfError "Data Protector Cell Manager ${CELL_SERVER} not responding to ping."
```

Comment:
Thank you for improving the wording to be more professional.

In general language that makes a computer sound as if it was a human
is considered to be not so much professional and more important
when computers talk to their users as if the computer was a human
it could be perceived by the user as strange or even offending language
in particular when things go wrong and the computer behaves patronizing.

So in general words like `please`, `sorry`, or even `congratulations` should be avoided.
Also words like `just`, `simple`, `easy` can be perceived as dismissive/patronizing:
```
Sorry, but you made a typo so your command failed.
Simply enter it correctly to get your job easily done.
Just read the fine manual for more information.
...
Congratulations! You finally made it.
```
;-)